tests/test_tmp_win.py

```diff
@@ -0,0 +1,55 @@
+import stock_management.tmp_win_utils as tmp_win_utils
+import time
+import sys
+
```

Comment:
Have a careful look to how tests are defined here. 
Therere are 2 tests.
The first is clicking the button `confirm` and checking that the desired "exit win Test: message is printed in the terminal.

The second is filling the input test widget with "pippo" , clicking the button "ok", and checking the message printed in the terminal.